microshift.sh

```diff
@@ -101,6 +101,8 @@ version = "*"
 EOF
     sed -i 's/redhat/core/g' scripts/image-builder/config/kickstart.ks.template
     sed -i "/--bootproto=dhcp/a\network --hostname=api.${SNC_PRODUCT_NAME}.${BASE_DOMAIN}" scripts/image-builder/config/kickstart.ks.template
+    sed -i 's/clearpart --all --initlabel/clearpart --all --initlabel --disklabel gpt/g' scripts/image-builder/config/kickstart.ks.template
+    sed -i "/clearpart --all/a\part biosboot --fstype=biosboot --size=1" scripts/image-builder/config/kickstart.ks.template
```

Comment:
Why the `\` before `part`?

Reply:
Just to explicitly separate out `/a` (append part).